app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post
+
+  def update_the_comment_counter
+    post.update(commentscounter: post.commentscounter + 1)
+  end
```

Comment:
- [ ] Here we can also implement the rails active callback `after_save` method as methioned in the post model